src/Magnum/Platform/Sdl2Application.cpp

```diff
@@ -100,8 +100,6 @@ bool Sdl2Application::tryCreateContext(const Configuration& configuration) {
     SDL_GL_SetAttribute(SDL_GL_FRAMEBUFFER_SRGB_CAPABLE, configuration.isSRGBCapable());
     #endif
 
-    /** @todo Remove when Emscripten has proper SDL2 support */
-    #ifndef CORRADE_TARGET_EMSCRIPTEN
     /* Set context version, if user-specified */
     if(configuration.version() != Version::None) {
```

Comment:
This line doesn't compile for me:

```
../src/Magnum/Platform/Sdl2Application.cpp:104:22: error: no member named 'version' in 'Magnum::Platform::Sdl2Application::Configuration'
    if(configuration.version() != Version::None) {
```

I force-pushed a rebased version on your branch, waiting for the CI to say what's up. Can you look into it after?

Also, while at it, there's also `MouseEvent::clickCount()` that's disabled on Emscripten currently (shouldn't be anymore if using SDL2) and the `isTextInputActive()` function has a probably-no-longer-necessary Emscripten workaround in it (you have to verify that the linker doesn't fail).

Sorry for the premature "no other comment" comment, then. These are things that can be seen only after I pull locally, not from the diff :D

Reply:
Alright, thanks for the review! I will find some time today to finish that up 👍